helm_deploy/cla-public/values-staging.yaml

```diff
@@ -15,7 +15,7 @@ ingress:
 
 envVars:
   BACKEND_BASE_URI:
-    value: "http://cla-backend-app.laa-cla-backend-staging.svc.cluster.local"
+    value: "http://cla-backend-app.laa-cla-backend-uat.svc.cluster.local"
```

Comment:
Note: It made sense for me to leave this to staging for staging - so we can test on staging easily. But happy to revert if we need it to connect to UAT instead.